posthog/utils.py

```diff
@@ -418,7 +419,7 @@ def render_template(
     context["posthog_js_uuid_version"] = settings.POSTHOG_JS_UUID_VERSION
 
     html = template.render(context, request=request)
-    return HttpResponse(html)
+    return patch_cache_control(HttpResponse(html), no_store=True)
```

Comment:
If we never want to cache, there's `@never_cache`

> from django.views.decorators.cache import never_cache
>
>
> @never_cache
> def myview(request): ...

Reply:
Code quality checks are right here.

One more thing: I believe we should only prevent caching for logged-in requests, i.e. `not request.user.is_anonymous`. So something like this:
```suggestion
    response = HttpResponse(html)
    if not user.request.is_anonymous:
        patch_cache_control(response, no_store=True)
    return response
```